packages/eas-cli/src/graphql/queries/PublishQuery.ts

```diff
@@ -0,0 +1,34 @@
+import publishGraphQL from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+
+const PublishQuery = {
```

Comment:
Putting all GraphQL queries in the `src/graphql/queries` folder and mutations in `src/graphql/mutations` folder was a pattern we tried initially, but abandoned it. It would be better to put the queries inside (or near) the modules where they're used.

Reply:
OK, I will do that when landing the following (3/4) stacked PR https://github.com/expo/eas-cli/pull/147 to make rebasing simpler.